### DISCUSSION TITLE
Termux & Repositories

### DISCUSSION THREAD

**Discussion by cb59601** [termux/termux-packages#13294] [General]
Hey guys back with another NOOB question for you. I'm trying to find where my local repositories are kept. I'm having some problems with apt and I'm wondering if it is a repository issue. Any thoughts or comments would be great thanks......

Happy Coding

**Comment by sylirre** [comment]
`$PREFIX/etc/apt/sources.list` - for primary repository.

Files in `$PREFIX/etc/apt/sources.list.d` for additional repositories.

Usually you need to use only `termux-change-repo` for switching to a working mirror.
